mlflow/R/mlflow/R/databricks-utils.R

```diff
@@ -161,7 +161,7 @@ mlflow_get_run_context.mlflow_databricks_client <- function(client, experiment_i
     } else {
       NA
     }
-    if (!is.na(job_info) && !is.na(job_info$job_id)) {
+    if (!all(is.na(job_info)) && !is.na(job_info$job_id)) {
```

Comment:
Can we update the R version to 4.3.0 to make sure this line works fine with it?

https://github.com/mlflow/mlflow/blob/ed44d8e33cd087a3881a61d3e6107b2689d9da40/.github/workflows/r.yml#L39-L42

Reply:
+1

Reply:
There are some failures in tests on time duration output format. I can change the tests but is it desired? @harupy @WeichenXu123 
```
── Failure ('test-tracking-runs.R:206:3'): mlflow_log_metric() rounds step and timestamp inputs ──
mlflow_get_metric_history("timestamp_metric")$timestamp (`actual`) and purrr::map(round(timestamp_inputs), mlflow:::milliseconds_to_date) (`expected`) don't have the same values.
* Only in `actual`:   1970-01-01 00:01:06.796, 1970-01-01 00:01:29.372, 1970-01-01 00:01:24.502, 1970-01-01 00:01:04.028, 1970-01-01 00:00:17.371, 1970-01-01 00:00:47.979, 1970-01-01 00:00:42.036, 1970-01-01 00:00:15.319, 1970-01-01 00:00:39.544, 1970-01-01 00:01:39.184
* Only in `expected`: 66.796, 89.372, 84.502, 64.028, 17.371, 47.979, 42.036, 15.319, 39.544, 99.184

── Failure ('test-tracking-runs.R:259:3'): mlflow_log_metric() with step produces expected metric data ──
metric_history_1$timestamp (`actual`) and purrr::map(c(300, 100, 200), mlflow:::milliseconds_to_date) (`expected`) don't have the same values.
* Only in `actual`:   1970-01-01 00:00:00.3, 1970-01-01 00:00:00.1, 1970-01-01 00:00:00.2
* Only in `expected`: 0.3, 0.1, 0.2

── Failure ('test-tracking-runs.R:439:3'): mlflow_log_batch() works ────────────
metrics$timestamp (`actual`) and purrr::map(c(200, 300, 400, 500, 600), mlflow:::milliseconds_to_date) (`expected`) don't have the same values.
* Only in `actual`:   1970-01-01 00:00:00.4, 1970-01-01 00:00:00.3, 1970-01-01 00:00:00.2, 1970-01-01 00:00:00.5, 1970-01-01 00:00:00.6
* Only in `expected`: 0.2, 0.3, 0.4, 0.5, 0.6

── Failure ('test-tracking-runs.R:453:3'): mlflow_log_batch() works ────────────
metric_history$timestamp (`actual`) and purrr::map(c(100, 200), mlflow:::milliseconds_to_date) (`expected`) don't have the same values.
* Only in `actual`:   1970-01-01 00:00:00.1, 1970-01-01 00:00:00.2
* Only in `expected`: 0.1, 0.2
```

Reply:
@jinzhang21 Let's fix them :)

Reply:
Found R 4.3.0 contains changes related to dates and times:
In https://cran.r-project.org/doc/manuals/r-release/NEWS.html